lib/index.js

```diff
@@ -106,6 +130,7 @@ exports.Model = class SchwiftyModel extends Model {
 
     parseValidationError(err) {
```

Comment:
Why is it strange?  Doesn't seem strange to me!  If it were static you'd just call `SchwiftyModel.parseValidationError()`.